Octokit.Tests/Clients/ReactionsClientTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using NSubstitute;
+using Octokit;
+using Octokit.Tests;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+public class ReactionsClientTests
+{
+    public class CommitComments
+    {
+        public class TheGetAllMethod
+        {
+            [Fact]
+            public async Task RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new ReactionsClient(connection);
```

Comment:
Ohh. That's right. No good Name for a test. I will split tests into separate classes.